tests/resources/inputs/SNAPInstPrm.json

```diff
@@ -24,5 +24,7 @@
     "delThWithGuide": 6.40e-3,
     "alpha": 0.1,
     "beta_0": 0.02,
-    "beta_1": 0.05
+    "beta_1": 0.05,
```

Comment:
make sure Malcolm didn't delete this line in the One True SNAPInstPrm.json

Reply:
the one on SNS has been updated correctly